src/app/backend/apihandler.go

```diff
@@ -279,6 +284,20 @@ func (apiHandler *ApiHandler) handleEvents(request *restful.Request, response *r
 	response.WriteHeaderAndEntity(http.StatusCreated, result)
 }
 
+// Handles get Replica Set Pods metrics.
+func (apiHandler *ApiHandler) handleReplicaSetPodsMetrics(request *restful.Request,
```

Comment:
Hmm... Why do we need a new api handler? Shouldn't the metrics be included in replicasetdetail response?